assets/embauche.html

```diff
@@ -107,177 +107,179 @@
 	<details id="taxes" class="js-only">
 		<summary>Cotisations détaillées</summary>
 
-		<table>
-			<caption>Cotisations employeur</caption>
+		<div class="tables">
```

Comment:
You're right for the open state, but it also fails when closed : the css rule `.SGMAPembauche table {display:inline-block}` takes precedence over `details > * {display: none}`, displaying the details content...